fdbclient/admin_client.go

```diff
@@ -229,13 +235,13 @@ func (client *cliAdminClient) GetStatus() (*fdbv1beta2.FoundationDBStatus, error
 	if client.useClientLibrary {
 		// This will call directly the database and fetch the status information
 		// from the system key space.
-		return getStatusFromDB(client.Cluster)
+		return getStatusFromDB(client.Cluster, client.log)
 	}
 	contents, err := client.runCommand(cliCommand{command: "status json"})
 	if err != nil {
 		return nil, err
 	}
-	log.V(1).Info("Fetched status JSON", "contents", contents)
```

Comment:
Could we get rid of this or increase the level to debug?

Reply:
This was already at `debug` level, it was just ignored because we didn't passed a configured logger (see the example log output).